nasa-gibs/worldview
# WV-2423: Update to webpack 5

## Description

Updates webpack and many other dependencies to the latest version

## How To Test

* Update dependencies: `npm ci`
* Confirm you can build and serve the app in watch mode:
  * `npm run watch:debug:devtools`
* Confirm you can build the app for prod and serve locally
  * `npm run build:prod`
  * `npm run start`
